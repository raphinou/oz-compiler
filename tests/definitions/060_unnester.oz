% Unnesting of variable assignment in declarations of a local and in proc argument
local
   A = 5-2+(8+4)*2
   B = A*2+4
in
   {Show A}
   {Show B}
   {Show 2*A+3*B}
end
