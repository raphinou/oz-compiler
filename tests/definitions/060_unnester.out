27
58
228
